flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/CatalogManager.java

```diff
@@ -888,7 +899,7 @@ public Set<String> listSchemas() {
      * <p><b>NOTE:</b>It is primarily used for interacting with Calcite's schema.
      *
      * @param catalogName filter for the catalog part of the schema
-     * @return list of schemas with the given prefix
+     * @return set of schemas with the given prefix
```

Comment:
This change seems unrelated to models.  I wonder if we could pull them out into a purely cosmetic PR?